src/main/resources/config.yml

```
@@ -125,6 +125,57 @@ web:
+    plugin-api: true
+    pdc-scan: true
+    custom-model-data: true
+    lore-pattern: true
+    inventory-scan: true
+    interaction-detect: true
+  excluded-namespaces:
+    - minecraft
+    - aureleconomy
+
+
+custom-items:
```

YAMLlint (1.38.0)

[error] duplication of key "custom-items" in mapping

(key-duplicates)